extension/js/background.js

```diff
@@ -83,7 +83,7 @@ async function makeRelayAddressForTargetElement(info, tab) {
 if (browser.menus) {
   browser.menus.create({
     id: "fx-private-relay-generate-alias",
-    title: "Generate Email Alias",
+    title: "Generate New Alias",
```

Comment:
Q: Why is this "Generate New Alias" (uppercases), but "Generate new alias" above on extension/js/add_input_icon.js#L133 (and L190)?

Reply:
Wasn't my decision but I think because other context menu items are also capitalized.